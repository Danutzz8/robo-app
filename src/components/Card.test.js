import { shallow, mount, render } from 'enzyme';
import { create } from "react-test-renderer";
import React from'react';
import Card from './Card';

// most used 90% will be shallow the other 2 mount/render will happen in a domAPI or browser
//====================================================================================
// it('expect to render Card component', () => {
//     expect(shallow(<Card/>).length).toEqual(1)
// });

it('expect to render Card component', () => {
    expect(shallow(<Card/>)).toMatchSnapshot()
});

// it("renders correctly", () => {
//     expect(shallow(<Card/>)).toMatchSnapshot(); // check if a component renders correctly
//   });