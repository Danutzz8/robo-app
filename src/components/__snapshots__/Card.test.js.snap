// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card component 1`] = `ShallowWrapper {}`;

exports[`renders correctly 1`] = `ShallowWrapper {}`;
